README.md

```diff
@@ -29,6 +29,8 @@ The main purpose of the project [Hello-Microverse](https://sheylapozo.github.io/
 
```

Comment:
- [ ] [OPTIONAL] Your live demo is broken, please fix it, or remove this section.

Reply:
fixed it